services/dynamic/dynamic-json.tester.js

```diff
@@ -103,7 +103,7 @@ t.create('JSON from url | invalid url')
 
 t.create('JSON from url | user color overrides default')
   .get(
-    '.json?url=https://github.com/badges/shields/raw/master/package.json&query=$.name&colorB=10ADED&style=_shields_test'
+    '.json?url=https://github.com/badges/shields/raw/master/package.json&query=$.name&color=10ADED&style=_shields_test'
```

Comment:
This has nothing to do with the service and really should be a unit test of BaseService.